config/pickit.ini

```diff
@@ -43,7 +42,7 @@ misc_greater_healing_potion=0
 misc_greater_mana_potion=0
 
 ;misc
-misc_gold=0
+misc_gold=1
```

Comment:
If you make misc_gold default 1 I would also make gold_trav_only param in [car] section default 1